egs2/dral/s2st1/requirements.txt

```diff
@@ -0,0 +1,5 @@
+backoff==2.2.1
```

Comment:
consider add a check at `local/path.sh` to ensure that the requirements are fulfilled (that's also the reason that we have a local/path.sh for every recipe.